File: BlogClient/src/app/Models/Posts/CreatePostRequestModel.ts
import { BasePostViewModel } from "./BasePostViewModel";

export class CreatePostRequestModel extends BasePostViewModel {
    constructor(
        title: string, 
        content: string, 
        excerpt: string, 
        isPublished: boolean, 
        categoryId: number, 
        tagId: number
    ) {
        super(title, content, excerpt, isPublished, categoryId, tagId);
    }
}

File: BlogApp/BlogApp.ViewModels/RequestModels/Posts/CreatePostRequestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BlogApp.ViewModels.RequestModels.Posts
{
    public class CreatePostRequestModel
    {

        public string Title { get; set; }
        public string Content { get; set; }
        public string Excerpt { get; set; }
        public bool IsPublished { get; set; }
        public int CategoryId { get; set; }
        public int TagId { get; set; }
    }
}
